### PR TITLE
[READY] Autogenerate locales file before build (#80). This removes the hardco…

### LINKED ISSUE #80: Chore / Cleanup: Move Locales list out of Toolbar script tag
Right now, the list of locales for the site menu exists directly in `Toolbar.vue` like so:

```
export default {
  data: () => ({
    drawer: null,
    locales: [
      { title: "العربية", value: "ar" },
      { title: "中文", value: "zh_CN" },
      { title: "Nederlands", value: "nl" },
      { title: "English", value: "en" },
      { title: "Filipino", value: "fil" },
      { title: "Français", value: "fr" },
      { title: "Deutsch", value: "de" },
      { title: "Bahasa Indonesia", value: "id" },
      { title: "Bahasa Melayu", value: "ms" },
      { title: "italiano", value: "it" },
      { title: "日本語", value: "ja" },
      { title: "한국어", value: "ko" },
      { title: "नेपाली", value: "ne" },
      { title: "Português", value: "pt" },
      { title: "Spanish", value: "es" },
      { title: "ไทย", value: "th" },
      { title: "Tiếng Việt", value: "vi" },
    ],
  }),
```

We should move this into the `data/` directory as JSON and import here to separate data and logic.

### PR DESCRIPTION
**Pull request by renderf0x**
…ded lists in Toolbar and nuxt.config.js files

This change depends on the `file` key being added to the localization repo, [per this PR](https://github.com/ourjapanlife/findadoc-localization/pull/39/files).

### Changes

Dynamically load language list instead of static lists in config files.

This PR has been updated to use the file generated in the localization repo as a source, instead of generation in this package.

### How to test

1. Pull, then `yarn build; yarn run` or `yarn dev`
2. Browse around and switch languages in the app - this should work exactly as before with no changes in functionality.

### Screenshots

スクリーンショット 2021-06-20 15 32 19

### CODE REVIEW

File: scripts/generate_languages_file.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const fs = require('fs');
+const path = require('path');
+const projectRootPath = path.resolve(__dirname, '../');
+const i18nPath = path.resolve(projectRootPath, 'i18n/locales');
+const outputFilePath = path.join(projectRootPath, 'languages.json')
+
+
+const languages = []
+
+fs.readdirSync(i18nPath).forEach(file => {
```
[the remark this answers is not in the document]
Comment (Ermish) [reply scripts/generate_languages_file.js:24]: I was thinking #2 here. We can't do anything with the error, but we can at least make it easier to read/debug if it does happen

[the remark this answers is not in the document]
Comment (renderf0x) [reply scripts/generate_languages_file.js]: Just obviated this =). After speaking with @ann-kilzer , we've decided to go with a flat config file in the loc repo, so this has been simplified to simply read that file.

It's waiting on one change in the localization package ([PR](https://github.com/ourjapanlife/findadoc-localization/pull/39/files)), which is why it's not starting up here. Will update here when that is through.

**Comment by renderf0x** [comment]
Ok, ready to go now. It's greatly simplified due to the changes made in localization

**Review by Ermish — APPROVED** [review]
LGTM!

**Pull request merged**